sbi/simulators/linear_gaussian.py

```diff
@@ -24,6 +24,42 @@ def get_true_posterior_samples_linear_gaussian_mvn_prior(
     return c * mean + std * torch.randn(num_samples, dim)
 
 
+def get_true_posterior_log_prob_linear_gaussian_n_prior(
+    observation: torch.Tensor, std=1.0
+):
+    """
+    Get the ground truth density when using just a single dimension.
+
+    Returns: torch.distributions.Normal
```

Comment:
One can specify the return type on the signature:
> def get_true_posterior_log_prob_linear_gaussian_n_prior(
    observation: torch.Tensor, std: **float**=1.0) ->torch.distributions.Distribution:
and give the return type in more informal language (e.g. a Normal distribution).

Also a kind reminder that PEP257 standard recommends to start the docstring with 'Returns...'